src/ConservationChecker.jl

```diff
@@ -16,193 +16,184 @@ using UnPack
 using Plots
 using ClimaAtmos: RRTMGPI
 using ClimaLSM
-using ..Utilities: CoupledSimulation, swap_space!
-using ..Interfacer: SurfaceStub
+using ClimaCoupler.Utilities: CoupledSimulation, swap_space!
+import ClimaCoupler: Interfacer
```

Comment:
I think Charlie has said it's better to use `import ..Interfacer` since that makes sure we use the dev version of the package we're working on, rather than potentially using the latest release of main. But there are a lot of places we aren't doing that, so maybe we could have another PR that goes through and changes that.

Reply:
Sounds good. When developing locally, I usually use `Pkg` to call `dev` on the entire local Coupler package, but happy to change to this (as a separate issue), if that makes development easier for you. :)